js/scrollspy.js

```diff
@@ -99,7 +99,7 @@
     for (i = offsets.length; i--;) {
       activeTarget != targets[i]
         && scrollTop >= offsets[i]
-        && (!offsets[i + 1] || scrollTop <= offsets[i + 1])
+        && (offsets.length <= i + 1 || scrollTop <= offsets[i + 1])
```

Comment:
Perhaps `offsets[i + 1] != undefined` might be better? Would be interested to hear other folks' opinions.